tools/reviewer/service/tests/CodeReviewServiceGetDiffFilesTest.java

```diff
@@ -0,0 +1,450 @@
+    String[] args = {
+      "init", aaBaseFolder,
+      "--startupos_repo", initialRepoFolder,
+    };
+    initCommand.run(args);
+  }
+
+  private void createAaWorkspace(String name) {
+    WorkspaceCommand workspaceCommand = component.getWorkspaceCommand();
+    String[] args = {"workspace", "-f", name};
+    workspaceCommand.run(args);
+    repoPath = fileUtils.joinPaths(getWorkspaceFolder(TEST_WORKSPACE), "startup-os");
+    repo = gitRepoFactory.create(repoPath);
+    repo.setFakeUsersData();
+    repo.switchBranch("D" + DIFF_ID);
+  }
+
+  private void createBlockingStub() throws IOException {
+    // Generate a unique in-process server name.
+    String serverName = InProcessServerBuilder.generateName();
+    server =
+        InProcessServerBuilder.forName(serverName)
+            .directExecutor()
+            .addService(codeReviewService)
+            .build()
+            .start();
+    channel = InProcessChannelBuilder.forName(serverName).directExecutor().build();
+    blockingStub = CodeReviewServiceGrpc.newBlockingStub(channel);
+  }
+
+  private String joinPaths(String first, String... more) {
+    return FileSystems.getDefault().getPath(first, more).toAbsolutePath().toString();
+  }
+
+  private String getWorkspaceFolder(String workspace) {
+    return joinPaths(aaBaseFolder, "ws", workspace);
+  }
+
+  private void writeFile(String contents) {
+    writeFile(TEST_FILE, contents);
+  }
+
+  private void writeFile(String filename, String contents) {
+    fileUtils.writeStringUnchecked(
+        contents, fileUtils.joinPaths(getWorkspaceFolder(TEST_WORKSPACE), "startup-os", filename));
+  }
+
+  private Protos.DiffFilesResponse getResponse() {
+    Protos.DiffFilesRequest request =
+        Protos.DiffFilesRequest.newBuilder()
+            .setWorkspace(TEST_WORKSPACE)
+            .setDiffId(DIFF_ID)
+            .build();
+
+    return blockingStub.getDiffFiles(request);
+  }
+
+  private com.google.startupos.common.repo.Protos.BranchInfo getExpectedBranchInfo() {
+    return com.google.startupos.common.repo.Protos.BranchInfo.newBuilder()
+        .setDiffId(DIFF_ID)
+        .setRepoId(REPO_ID)
+        .addCommit(
+            com.google.startupos.common.repo.Protos.Commit.newBuilder()
+                .setId(fileInHeadCommitId)
+                .build())
+        .addCommit(
+            com.google.startupos.common.repo.Protos.Commit.newBuilder()
+                .setId(testFileCommitId)
+                .addFile(
+                    com.google.startupos.common.repo.Protos.File.newBuilder()
+                        .setFilename("test_file.txt")
+                        .setWorkspace("ws1")
+                        .setRepoId(REPO_ID)
+                        .setCommitId(testFileCommitId)
+                        .setFilenameWithRepo("startup-os/test_file.txt")
+                        .build())
+                .build())
+        .build();
+  }
+
+  private Protos.DiffFilesResponse getExpectedResponseAddingCommit(
```

Comment:
I think it would be nicer to import `DiffFilesResponse` so you don't need the `Protos.` prefix.

Reply:
Done.